thorlcr/activities/join/thjoinslave.cpp

```diff
@@ -387,7 +390,10 @@ class JoinSlaveActivity : public CSlaveActivity, implements ILookAheadStopNotify
             {
                 unsigned bn=noSortPartitionSide()?2:4;
                 ActPrintLog("JOIN waiting barrier.%d",bn);
-                barrier->wait(false);
+                {
+                    BlockedActivityTimer t(slaveTimerStats, timeActivities);
```

Comment:
I realize there's BlockActivityTimer's in other stops already (via hash agg stream),

but .. stop() time in general is not being timed (as part of total cycles).. *may be it should be - but that's a separate question*. 
So that means, if blocked time is, then processCycles could be < blockedCycles ( and result in warnings from queryLocalCycles)

If any act measures blocked in stop(), it would mean it needs to consider stop() in the calculation in general which it doesn't. 
Can you remove this (and separately other Blocked time considerations in other stop'd contexts) and open a separate JIRA to revisit measuring time in general in activities stop()'s.